docs/SEEEDSTUDIO_WIO_E5.md

```diff
@@ -83,18 +83,20 @@ The first time that you flash the mLRS firmware to either the E5 Mini or E5 Grov
 - Connect your ST-Link to the 3V3, GND, SWD, and SWCLK pins on the board
     - The SWD and SWCLK pins are labeled 'DIO' and 'CLK' respectively
     - Some ST-Link boards provide power and some do not - ensure the E5 is powered
-- Launch STM32CubeProgrammer and select ST-Link as the connection method, click connect
+    - While applying power also hold the boot button or ground out the boot pad for the Wio E5-Grove, this will ensure initial connection with STM32CubeProgrammer
+- Launch STM32CubeProgrammer and select ST-Link as the connection method, click connect 
 - From the menu on the left select the OB (Option Bytes) tile
 - From the Read Out Protection section, change to AA, select Apply
     - This will erase the factory firmware
 - From the menu on the left select the Download tile
+- Before loading new firmware ensure you do a full chip erase, this will allow the modules to be bound initially without the need to use the other binding methods
```

Comment:
This seems extraneous to me as disabling readout protection will in fact do a full chip erase.

Reply:
Even when doing the readout protection they would not bind. I just went through and did a full chip erase on all and had no problems binding. If I didn't change the readout I don't think I would have been able to program them correct?

Reply:
Hmm, any chance this was on the hardware I sent?  If so, readout protection would've already been disabled and then I can see that disabling wouldn't have done a chip erase.  Regardless, I suppose there's no harm in doing a chip erase after disabling readout protection so no issues keeping.

Reply:
no this was on the stuff I got from seeedstudio. I had a set that I manually did a full chip erase on and had no issue but a few others would not bind as I didn't do full chip erase. I have another set coming in the mail so maybe we just leave until I can verify the removing readout protection does in fact allow initial binding?

Reply:
It would be excellent if you could test with your new set :). To me, having readout protection in the first place is rather odd for what is supposed to be a development board.

Reply:
I guess I would tone it down a bit and e.g. say something like "Before loading new firmware it can be advisable to do a full chip erase. This will reset all parameters to factory default and ensures that the unit is in a defined state."

note that the underlying claim, that any tx and rx will behave as being already bound and will just connect when both are in virgin state, is not necessarily correct! 
If for cases where they should have been bound & connect after first flashing but they didn't, when we rather should work out the issue and correct the code.